app/routes/public.js

```diff
@@ -5,7 +5,7 @@ module.exports = [{
   path: '/robots.txt',
```

Comment:
If we've removed public from the path, then it makes sense to rename this file `static.js` for consistency.